Add plugin: Live Background

remememe opened the pull request:
# I am submitting a new Community Plugin.

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/remememe/Live-Wallpaper

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [x]  macOS
  - [x]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
[private ensureWindowControlsAccessibility() {](https://github.com/remememe/Live-Wallpaper/blob/ace3a92d2d8e39f7975f290670d059f8e60824d2/src/main.ts#L102-L131), [private applyTransparentBackgroundStyles() {](https://github.com/remememe/Live-Wallpaper/blob/ace3a92d2d8e39f7975f290670d059f8e60824d2/src/main.ts#L191-L238)
This should be in the `styles.css` file.

[.setName('Wallpaper Source')](https://github.com/remememe/Live-Wallpaper/blob/ace3a92d2d8e39f7975f290670d059f8e60824d2/src/main.ts#L330), [.setName('Wallpaper Opacity')](https://github.com/remememe/Live-Wallpaper/blob/ace3a92d2d8e39f7975f290670d059f8e60824d2/src/main.ts#L340), [.setName('Blur Radius')](https://github.com/remememe/Live-Wallpaper/blob/ace3a92d2d8e39f7975f290670d059f8e60824d2/src/main.ts#L360), [.setName('Layer Position (Z‑Index)')](https://github.com/remememe/Live-Wallpaper/blob/ace3a92d2d8e39f7975f290670d059f8e60824d2/src/main.ts#L400), [.setName('Layer Position (Z‑Index)')](https://github.com/remememe/Live-Wallpaper/blob/ace3a92d2d8e39f7975f290670d059f8e60824d2/src/main.ts#L400)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI)

remememe commented:
Hi @joethei 
Thanks for the feedback! I've moved the styles to styles.css and updated the UI labels to use sentence case as suggested. Let me know if anything else needs adjustment.
